tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -1495,6 +1495,24 @@ public function testClosureTypesContainingBackslash(): void
         ');
     }
 
+    public function testTypesConditional(): void
```

Comment:
Name of this test does not represent tested cases - there are no conditionals.

Reply:
fixed 👍